cmd/foo/main.go

```diff
@@ -1,7 +1,15 @@
 package main
 
-import "fmt"
+import (
+	"errors"
+	"fmt"
+)
 
 func main() {
 	fmt.Println("foo")
 }
+
```

Comment:
exported function Bar should have comment or be unexported

https://golang.org/wiki/CodeReviewComments#doc-comments